services/metadactyl-clj/src/metadactyl/service/apps.clj

```diff
@@ -30,8 +28,8 @@
 
 (defn- authorization-redirect
   [server-info username state-info]
-  (throw+ {:error_code ce/ERR_TEMPORARILY_MOVED
-           :location   (authorization-uri server-info username state-info)}))
+  (throw+ {:type       :clojure-commons.exception/temporary-redirect
+           :location (authorization-uri server-info username state-info)}))
```

Comment:
The prior alignment of values in this map should probably either be preserved or all keys should use a single space.